app/controllers/application_controller.rb

```diff
@@ -53,57 +44,16 @@ def prepare_menu
     end
   end
 
-  def set_sentry_request_context
-    Raven.extra_context(params: params.to_unsafe_h, url: request.url)
-  end
-
   private
 
-  def logout
-    flash_notice = params[:autologout] ? t('session.destroy.expired') : flash[:notice]
-    jumpto = params[:jumpto]
-    reset_session
-    session[:jumpto] = jumpto
-    flash[:notice] = flash_notice
-
-    redirect_to session_new_path
-  end
-
-  def handle_pending_recrypt_request
-    if pending_recrypt_request?
-      pending_recrypt_request_message
-      logout
-    end
-  end
-
-  def check_if_user_logged_in
-    if current_user.nil?
-      session[:jumpto] = request.parameters
-      redirect_to session_new_path
-    end
-  end
-
-  def validate_user
-    handle_pending_recrypt_request
-    check_if_user_logged_in
+  def set_sentry_request_context
```

Comment:
sentry ? not part of this feature, is it ?

Reply:
It's not, the method was moved while cleaning up the ApplicationController